main/webapp/modules/core/styles/dialogs/clustering-functions-dialog.css

```diff
@@ -0,0 +1,76 @@
+/*
+
+Copyright 2010, Google Inc.
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are
+met:
+
+ * Redistributions of source code must retain the above copyright
+notice, this list of conditions and the following disclaimer.
+ * Redistributions in binary form must reproduce the above
+copyright notice, this list of conditions and the following disclaimer
+in the documentation and/or other materials provided with the
+distribution.
+ * Neither the name of Google Inc. nor the names of its
+contributors may be used to endorse or promote products derived from
+this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,           
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY           
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+ */
+.clustering-functions-table {
+    width: 652px;
+    table-layout: fixed;
+}
+
+#clustering-functions-heading {
+    background: #ddd;
+    font-weight: bold;
+    padding: 6px 12px;
+}
+
+.clustering-functions-container {
+    min-height: 67px;
+    max-height: 240px;
+    overflow: auto;
+}
+
+.clustering-functions-container td {
+    border: 1px solid #ddd;
+    padding: 8px 12px;
+    text-align: left;
+    vertical-align: middle;
+    word-wrap: break-word;
+}
+
+.clustering-functions-container td button {
+    margin-right: 5px;
+    padding: 2px 7px;
+}
+
+.main-text {
+    font-size: 15px;
+    color: #333;
+}
+
+.sub-text {
```

Comment:
Did you intentionally not prefix the two CSS classes above?
Given that they are pretty generic I would say it's not unlikely that they match something else outside of the dialog, for instance introduced by an extension.
The fact that our CSS is split into many different files sadly doesn't mean that we have any isolation.

Instead of changing the class name you could also consider making the CSS selector more specific, such as `#clustering-functions-tabs .main-text`.